frontend/src/container/BillingContainer/BillingUsageGraph/utils.ts

```diff
@@ -0,0 +1,91 @@
+import { isEmpty, isNull } from 'lodash-es';
+import { MetricRangePayloadProps } from 'types/api/metrics/getQueryRange';
+
+export const convertDataToMetricRangePayload = (
+	data: any,
+): MetricRangePayloadProps => {
+	const emptyStateData = {
+		data: {
+			newResult: { data: { result: [], resultType: '' } },
+			result: [],
+			resultType: '',
+		},
+	};
+	if (isEmpty(data)) {
+		return emptyStateData;
+	}
+	const {
+		details: { breakdown = [] },
+	} = data || {};
+
+	if (isNull(breakdown) || breakdown.length === 0) {
+		return emptyStateData;
+	}
+
+	const payload = breakdown
+		.sort((a: any, b: any) => a.timestamp - b.timestamp)
```

Comment:
according to the same response attached , breakdown is an array of objects which doesn't seem to have timestamp property, why are we sorting metrics/ logs / traces objects ? and they do not seem to contain this field as well

Reply:
yeah, forgot to remove this